COLLADAMaya/src/COLLADAMayaControllerExporter.cpp

```diff
@@ -555,15 +583,17 @@ namespace COLLADAMaya
                 SceneElement* foundElement = mDocumentExporter->getSceneGraph()->findElement ( pathes[i] );
 				if (foundElement != NULL && !result)
 				{
-					foundElement->addSkeletonURI(skeletonUri);
-					UppersceneElement = sceneElement;
+					
+					JointRootElement = searchJointRootElement(sceneElement);
+					COLLADASW::URI skeletonRootUri(visualSceneExporter->getSceneElementURI(JointRootElement));
+					foundElement->addSkeletonURI(skeletonRootUri);
```

Comment:
From Api Maya we get all influences joint, then i take the local root (from all those joints), then i go upward until it is a joint type (not a node) >> searchJointRootElement function.
So i have the maximum search space, then  i can add all node in joint array (controller) to include all bind pose.